tom_tns/tns_api.py

```diff
@@ -43,6 +43,15 @@ def default_authors():
         return settings.BROKERS.get('TNS', {}).get('default_authors', '')
 
 
+def group_names():
+    """ Returns the Group names list from settings, either from Hermes config or TNS config
+    """
+    if submit_through_hermes():
```

Comment:
I think this (and the functions above) breaks the description of the PR. These will take the Hermes credentials even if local TNS credentials exist.

Reply:
This is the only part that we might want to use TNS settings for to over-ride the Hermes settings if available. I could see people wanting to have a different TNS default author list than Hermes default author list...